frontend/src/metabase/query_builder/containers/QueryBuilder.jsx

```diff
@@ -326,6 +337,7 @@ function QueryBuilder(props) {
 
 export default _.compose(
   Bookmark.loadList(),
+  Timelines.loadList(timelineProps),
```

Comment:
Are we sure we want to block the QB rendering until the timelines are fetched?

Reply:
There is `loadingAndErrorWrapper: false`, it's not blocking